testing-daml-models/daml-scripts/5_daml_repl.md

```diff
@@ -0,0 +1,62 @@
+Instead of writing a script in a DAML source file, you can also connect to the ledger with the `DAML REPL` and run your commands interactively step by step. This is very helpful if you are trying to get a better feel for how your code works.
+
+First, kill the running sandbox in the first terminal (ctrl-c) and restart it to get back into a clean state.
+
+```
+daml sandbox .daml/dist/create-daml-app-0.1.0.dar
+```{{execute T1}}
+
+To start the `DAML REPL` run
+
+```
+cd create-daml-app
+daml repl --ledger-host=localhost --ledger-port=6865 .daml/dist/create-daml-app-0.1.0.dar
+```{{execute T2}}
+
+in the second terminal. After a few seconds you'll see the REPL prompt:
+
+```
+daml>
+```
+
+You can always exit the DAML REPL by pressing `ctrl-c`.
+
+At the REPL prompt you can issue an arbitrary DAML script command. As before we'll allocate a new party:
```

Comment:
Side note: The current UX for the REPL is kind of shitty. The only way to use it against an actual ledger is to use `partyFromText` since you do not want to allocate a fresh party here. Still not quite sure what the best solution here is. One option would be to have an overloaded strings instance for `Party` in the repl. Happy to hear suggestions!